### PR TITLE
Add delete sticker pack method

### PR DESCRIPTION
**Pull request by denieryd**
# Description

Add delete sticker pack method

# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: botx/client/stickers_api/delete_sticker_pack.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+class BotXAPIDeleteStickerPackResponsePayload(VerifiedPayloadBaseModel):
+    status: Literal["ok"]
+
+
+class DeleteStickerPackMethod(AuthorizedBotXMethod):
+    status_handlers = {
+        **AuthorizedBotXMethod.status_handlers,
+    }
```
Comment (Nimond) [inline botx/client/stickers_api/delete_sticker_pack.py:30]: You don't have to define it empty. It's set by default

File: botx/bot/bot.py [hunk 2]
```diff
@@ -1046,6 +1050,26 @@ async def create_sticker_pack(self, *, bot_id: UUID, name: str) -> StickerPack:
...
+        """
+
+        method = DeleteStickerPackMethod(
+            bot_id,
+            self._httpx_client,
+            self._bot_accounts_storage,
+        )
+
+        payload = BotXAPIDeleteStickerPackRequestPayload.from_domain(
+            sticker_pack_id=sticker_pack_id,
```
Comment (Nimond) [inline botx/bot/bot.py:1068]: In `botx.bot.bot:Bot` all methods use positional arguments (except `delete_sticker` and `send_smartapp_notification`). In a separate commit unify all Bot methods to kwargs

Comment (denieryd) [reply botx/bot/bot.py]: fixed

File: botx/client/stickers_api/delete_sticker_pack.py (hunk 1 above)
Comment (denieryd) [reply botx/client/stickers_api/delete_sticker_pack.py]: fixed

**Review by lxmnk — APPROVED** [review]

**Pull request merged**